README.md

```diff
@@ -29,6 +29,35 @@ Alternatively, if you already have JHipster installed, you can use this blueprin
 $ jhipster --blueprints micronaut
 ```
 
+## Using Docker
+
+Download the Dockerfile:
+
+```bash
+mkdir docker
+cd docker
+wget https://github.com/jhipster/generator-jhipster-micronaut/raw/master/docker/Dockerfile
```

Comment:
Not sure we should use `master` here. I am thinking about e.g. migration to jh7 where master will be most likely unstable. Wouldn't it be better to let point to a concrete tag so one uses a release version? (Pretty sure the users doing a docker build will be advanced users and know what they do, but you never know).

Reply:
ok sure . 

Good Initiative .

As , Soon as it is merged that url will become working